test/test_cli_integration.py

```diff
@@ -108,18 +109,24 @@ def delete_policy_from_string(self, policy):
 
         return output
 
-    def get_variable(self, *variable_ids):
+    def get_variable(self, variable_ids):
         return invoke_cli(self, self.cli_auth_params,
             ['variable', 'get', *variable_ids])
 
     def assert_set_and_get(self, variable_id):
         expected_value = uuid.uuid4().hex
-
         self.set_variable(variable_id, expected_value)
-        output = self.get_variable(variable_id)
+        output = self.get_variable([variable_id])
 
         self.assertEquals(expected_value, output)
 
+    def assert_json_equals(self, json1_str, json2_str):
+        json1 = json.loads(json1_str)
```

Comment:
please give it more meaningful names. `1`, `11`, `2`, etc isn't explanatory

Reply:
Sure